src/app/app.component.ts

```diff
@@ -1,10 +1,43 @@
-import { Component } from '@angular/core';
+import { Component } from "@angular/core";
+import {
+  faPencil,
+  IconDefinition,
+  faTrashCan,
+} from "@fortawesome/free-solid-svg-icons";
+import { mockedCoursesList } from "./shared/mocks/mocks";
 
 @Component({
-  selector: 'app-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  selector: "app-root",
```

Comment:
Is there a special reason to use double quotes instead of single quotes?

Reply:
No reason, it changed automatically somehow, I don't know why :D